.gitignore

```diff
@@ -53,6 +53,9 @@
 # Debug files
 scripts/**/*.json
 
+# Python cache
+scripts/templates/__pycache__/
```

Comment:
This dir can be generated in `scripts/`, too. Maybe generalize this to sth like `scripts/**/__pycache__/`?

Reply:
Done